src/atom_finder/constants.clj

```diff
@@ -9,5 +9,5 @@
 (def ag-repo (gitp/load-repo ag-path))
 
 (def root (tu (resource-path "logic-as-control-flow.c")))
-(def big-root (tu (expand-home "~/opt/src/github-top-c/php-src/ext/sqlite3/libsqlite/sqlite3.c")))
+;(def big-root (tu (expand-home "~/opt/src/github-top-c/php-src/ext/sqlite3/libsqlite/sqlite3.c")))
```

Comment:
Sorry! I shouldn't have left that there! Incidentally this is a very good file to test on, if you get the chance to download php-src, it's pretty valuable.